Tabnine/src/main/java/com/tabnine/statusBar/TabnineStatusBarWidget.java

```diff
@@ -62,17 +68,21 @@ public TabnineStatusBarWidget(@NotNull Project project) {
               update();
             });
 
-    CapabilityNotifier.Companion.subscribe(
-        capabilities -> {
-          if (capabilities.isReady()) {
-            boolean newForceRegistration = capabilities.isEnabled(Capability.FORCE_REGISTRATION);
+    CapabilitiesStateSingleton.getInstance()
```

Comment:
do all IJ windows share this singleton? or do they run in different processes?

Reply:
No, its per IJ process. Every IJ window is a separate process.

Reply:
so will these changes solve also the multiple windows issues?